backend/tests/test_search.py

```diff
@@ -128,3 +128,29 @@ def test_limit(client, db):
     response = json.loads(rv.data)
     assert len(response) == limit
     assert response[0]['upc'] == matches[0].upc
+
+
+def test_search_close(client, db):
+    for i in test_data.valid_items:
+        copy.deepcopy(i).save()
+
+    lat = 0
+    long = 0
+    miles = 10
+
+    rv = client.get('/search_gps?lat=' + str(lat) + "&long=" + str(long) + "&miles=" + str(miles))
+    response = json.loads(rv.data)
+    assert len(response) == 2
+
+
+def test_search_1000(client, db):
+    for i in test_data.valid_items:
+        copy.deepcopy(i).save()
+
+    lat = 0
+    long = 0
+    miles = 1000
+
+    rv = client.get('/search_gps?lat=' + str(lat) + "&long=" + str(long) + "&miles=" + str(miles))
+    response = json.loads(rv.data)
+    assert len(response) == 3
```

Comment:
Please do not hardcode the test conditions; if we add more test stores, the number may change from 3. Instead, programmatically determine the correct number of stores to return (by iterating through the test items and checking their distance.)

Reply:
Done